Resources/Prototypes/_Lust/Arrows/Arrows_implants.yml

```
@@ -0,0 +1,200 @@
+      implant: WallImplant
+
+#ИМПЛАНТ ПЕРЕГРУЗКИ
+
+- type: entity
+  parent: BaseSmiteAction
+  id: ActionSmiteImp
+  name: Overload
+  description: Point your cursor at a person and activate the overload, and any electrical devices will destroy the target.
+  noSpawn: true
+  components:
+  - type: Action
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)